src/components/sbb-link/sbb-link.tsx

```diff
@@ -17,43 +17,56 @@ import { i18nTargetOpensInNewWindow } from '../../global/i18n';
 export class SbbLink {
   /**
    * If set to true, the browser will
-   * show the download dialog on click.
+   * show the download dialog on click (optional).
    */
   @Prop() public download?: boolean;
 
-  /** The href value you want to link to */
-  @Prop() public hrefValue!: string;
+  /** The href value you want to link to (mandatory otherwise the link becomes a button)*/
+  @Prop() public href!: string;
 
   /**
    * The icon name we want to use,
    * choose from the small icon variants from
    * the ui-icons category from here
-   * https://lyne.sbb.ch/tokens/icons/.
+   * https://lyne.sbb.ch/tokens/icons/ (optional).
    * Inline variant doesn't support icons.
    */
   @Prop() public icon?: string;
 
   /**
    * Pass in an id, if you need to identify
-   * the link element.
+   * the link element (optional).
    */
   @Prop() public idValue?: string;
 
   /**
    * Decide whether the icon should get flipped
    * horizontally if the document writing mode
-   * is changed from ltr to rtl or vice versa.
+   * is changed from ltr to rtl or vice versa (optional).
    */
   @Prop() public iconFlip?: boolean;
 
+  /**
+   * If this is set to true an span element will be used
+   * instead of an anchor or a button
+   */
+  @Prop() public isStatic = false;
+
   /**
    * The icon can either be place before or after
    * the text.
    */
   @Prop() public iconPlacement: InterfaceLinkAttributes['iconPlacement'] = 'start';
 
-  /** The link text we want to visually show. */
-  @Prop() public text!: string;
+  /**
+   * The aria-label attribute text. (optional)
+   */
+  @Prop() public accessibilityLabel?: string;
```

Comment:
Could you also add the other accessibility properties:
```ts
/** This will be forwarded as aria-describedby to the relevant nested element. */
@Prop() public accessibilityDescribedby?: string;
/** This will be forwarded as aria-label to the relevant nested element. */
@Prop() public accessibilityLabel?: string;
/** This will be forwarded as aria-labelledby to the relevant nested element. */
@Prop() public accessibilityLabelledby?: string;
```